app/controllers/saved_pages_controller.rb

```diff
@@ -0,0 +1,72 @@
+class SavedPagesController < ApplicationController
+  include AuthenticatedApiConcern
+
+  before_action :check_saved_page_exists, only: %i[destroy show]
+
+  # GET /api/saved_pages
+  def index
+    @saved_pages = SavedPage.where(oidc_user_id: user.id)
```

Comment:
This doesn't need to be an instance variable, and also use `user.saved_pages` to make it clearer.

Reply:
much better thank you, have resolved and will force push once I've tidied up a few other bits.